widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -8535,6 +8535,50 @@ div.dijitDateTextBox.dijitTextBox .dijitReset.dijitRight.dijitButtonNode.dijitAr
   border-bottom: 1px solid #e0e0e0;
 }
 
+/* start for styling manage data sources dialog for safari browser */
+.safari #datasourceAdminDialog .gwt-ScrollTable {
+  --scrollbar-width: 0px;
+}
+
+.safari #datasourceAdminDialog .gwt-ScrollTable .headerWrapper .headerTable {
+  width: calc(100% - var(--scrollbar-width)) !important;
+  box-sizing: border-box;
+}
+
+.safari #datasourceAdminDialog .gwt-ScrollTable .headerWrapper {
+  background-color: #666666;
+  border: 1px solid #CCCCCC !important;
+  border-right: 1px solid #99BEDB;
+}
+
+.safari #datasourceAdminDialog .gwt-ScrollTable .headerTable tr > td {
+  border-width: 0;
+}
+
+.safari #datasourceAdminDialog .gwt-ScrollTable .headerTable tr > td:last-of-type {
+  border-left-width: 1px;
+}
+
+.safari #datasourceAdminDialog .gwt-ScrollTable :is(.headerWrapper, .dataWrapper) {
```

Comment:
Some of these rules are redundant and could be combined:
`selector1,
selector2,
selector3 {
styles
}`

Reply:
Can you clarify which ones? Not spotting.